docs/ballerina-by-example/examples/strings/strings.bal

```diff
@@ -10,12 +10,12 @@ function main (string[] args) {
     string s2 = statement.toLowerCase();
     io:println("ToLower: " + s2);
 
-    //Compares two strings, ignoring the case. Returns True if the strings are equal; false otherwise.
+    //Compares two strings, ignoring the case. Returns True if the strings are equal and False otherwise.
```

Comment:
it should be true and false (simple letters).. Original comment is also wrong :( (not "True", but should be "true")

Reply:
Fixed.